### ISSUE TITLE
Synchronize between the two remote viewers

### ISSUE THREAD

**Issue by jeau** [openseadragon/openseadragon#2422]
This may be a stupid question, but is there any way to synchronize the display between two remote viewers, on two different browsers?

I wonder that after reading the issue #1862 _Synchronize animations between the two viewers_.

Sometimes I use Meet, Zoom, Jitsi, and others to do demonstrations of [Adno](https://adno.app/en/), which uses Openseadragon. The display is not reactive and the image quality is lost.

**Comment by pearcetm** [comment]
I totally agree that sharing OpenSeadragon images by videoconferencing often doesn't work well. Synchronizing viewers across multiple browsers would be really cool! It isn't something that can be directly added to OpenSeadragon, though. It would require some server-side infrastructure of some kind to coordinate messages between different browsers, and to figure out which users need to be synced. Once a session is set up, coordinates and zoom information could be sent via websockets or the like, to implement synchronized viewing similar to how it was described in the issue you mentioned.

**Comment by jeau** [comment]
I just found this work by @superbland from @Cogapp who implemented this for [Storiiies](http://storiiies.cogapp.com/) broadcast mode and this fun [OpenCCTVdragon](http://opencctvdragon.cogapp.com/) project. I can't get Storiiies broadcast mode to work, although I have no problems getting OpenCCTVdragon to work. 

- http://opencctvdragon.cogapp.com/
- https://github.com/superbland/opencctvdragon/

**Comment by jeau** [comment]
Another example with Annona https://ncsu-libraries.github.io/annona/storyboard/#web-sockets

**Comment by iangilman** [comment]
@jeau @pearcetm Cool! This is a great topic. Neat to see so many people have been doing it!

Speaking of which, thank you for all the links... I've added a bunch of them here: https://github.com/openseadragon/site-build/issues/200 I suppose at some point someone should really get around to updating our "In the wild" page 😄
